pubsub/connection.go

```diff
@@ -190,7 +190,7 @@ func (c *connection) handleNewBloom(cmd *NewBloom) error {
 	if !cmd.IsParamsValid() {
 		return ErrInvalidFilterParam
 	}
-	filter, err := bloom.New(uint64(cmd.MaxElements), float64(cmd.CollisionProb), MaxBytes)
+	filter, err := bloom.New(int(cmd.MaxElements), float64(cmd.CollisionProb), MaxBytes)
```

Comment:
`cmd.MaxElements` is a `json.Uint64`. Are we concerned about this subtle functionality change? I suppose it's not a big concern since the pubsub server is deprecated

Reply:
If something larger than MaxInt is provided... it'll under flow and error. previously it would have errored due to having too many bytes. So I think it's fine